docker-install/README.md

```diff
@@ -43,8 +43,74 @@ After you complete the operating system specific portions, scroll to the bottom
 1. Download Docker Community Edition for Windows and follow the prompts.
 ![Windows 10 Download](screenshots/win10-01-getdocker.png)
 
+### Post-Docker-install steps using Kitematic
```

Comment:
I think we want this to stay in the RNA-seq portion, since it's specific to the RNA-seq image. However, I think "post-install" is probably a bad way to name it. I wonder if we could fix the confusion by using something like: "Setup the RNA-seq Docker Container".

If we ultimately have to combine these things to get folks to do them, we can go that route. However, it makes it harder to give instruction on methods that use different docker containers in the future (say, a ChIP-seq or exome-seq module). If we can avoid it and still deliver the training successfully then we would be better off avoiding it.

Reply:
I don't think changing the name will help with the confusion. The main thing I think that was confusing was for them to have to go to two different places for set-up instructions. I'm not sure what's a good solution. We don't want people to be lost on the first set of instructions we give them, but I get that a goal is to keep this modularized.

Reply:
I've proposed a name change`docker-install` folder to be named `set-up` or something that's not `docker-install`. I think that name is confusing since they've already installed Docker on the RNA-seq spot.

Reply:
Never mind. Changing back. I also am apparently confused on where things are. Never mind.